README.md

```diff
@@ -48,14 +48,65 @@ Setup development environment
 ```
 
 Config file
-------------------------------------
+----------------------------------------------------------------------------------------------------------------------
 `config.json` file should be located in the same directory as all source files. Sample configuration is included in `config.json.sample` file.
 
 Possible fields in config file:
-
+- `authUrl` - URL to keystone
+- `gohan` - gohan connection config, must contain object with following fields:
+  - `schema` - path to jsonschema
+  - `url` - URL to gohan server
+- `sidebar` - array of custom sidebar buttons, must contain objects with following fields:
+  - `path`
+  - `title` 
+  - `id`
+- `sidebarFavorites` - array of button id's to shows in the upper part of the sidebar
+- `sidebarCategories` - array of object to grouping buttons in sidebar menu categories, must contain objects with following fields:
+  - `name` - display name of category
+  - `id` - id of category
+  - `items` - array of button id's to shows in the category
 - `sidebarChildResources` - array of schema IDs of child resources which should be visible in sidebar menu
 - `routes` - config array for routing, must contain objects with following fields:
   - `path` - URL path to view
   - `viewClass` - name of React component related to the view (in `src/routes/componentList.js`)
   - `name` - name of route
   - `exact` - match path exactly, default `true`
+- `errorMessages` - map of error messages
+  - `tokenExpire` - token expired message
+- `addingRelationDialog` - array of ????
+- `pageLimit` - limit of the page size
+- `loginRequestTimeout` - time before token expiration, when token renew dialog should be shown in ms
+- `polling` - flag to enable polling
+- `pollingInterval` - interval of polling
+- `storagePrefix` - prefix to store token information (local and session storage)
+- `tableDefaultSortKey` - default sort key
+- `tableDefaultSortOrder` - default sort order
+- `title`- title of web page
+- `followableRelations` - flat to enable followable resource by reference
+- `useKeystoneDomain` - flag to enable use domain feature in keystone v3
```

Comment:
`flag to enable using domain feature in Keystone v3`
How about adding a link to Keystone doc?